lib/msprime.c

```diff
@@ -6568,25 +6444,10 @@ msp_set_simulation_model_beta(msp_t *self, double reference_size, double alpha,
     self->model.params.beta_coalescent.scalar = scalar;
```

Comment:
I know this is left over from before, but ``scalar`` isn't the most descriptive of names. Is there much point in actually storing this I wonder?

Reply:
I've dropped it from the model, and instead compute it on the fly in each of the rescaling functions. They've become a little longer as a result, but are ultimately still pretty simple.